npm run build 报错

Error: Cannot find module 'china-ex\生成字体子集.js;node'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "description": "中国制霸生成器",
   "main": "html/index.html",
   "scripts": {
-    "build": "node 生成字体子集.js;node 转译.js"
+    "build": "node 生成字体子集.js && node 转译.js"
   },
   "repository": {
     "type": "git",

